docusaurus/docs/Android/04-guides/04-theming.md

```diff
@@ -8,49 +12,76 @@ This doesn't apply the changes instantly, the view must be recreated for the cha
 
 Here are some examples of styling you can perform here:
 
+**Change avatar image style:**
```

Comment:
I'm not sure if it makes sense to list all possible transformers. I would keep only one or two (most important), add a link to the `TransformStyle` object and instead - show how it works in practice. For example:
1. Show `ChannelListView` XML with custom attributes
2. Show applied `TransformStyle`
3. Show the result
What do you think?

Reply:
Makes sense